content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Bias in User Research'
+Description: 'In user research, bias refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results.'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In [user research](https://www.codecademy.com/resources/docs/uiux/user-research), **bias** refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results.
+
+To obtain accurate, useful data, researchers must be aware of their own biases. Researchers can also work to prevent participants from displaying common biases by strategically designing their studies. When analyzing results, researchers should consider how bias may have impacted the data, especially if the data is self-reported.
+
+### Acquiescence Bias
+
+**Acquiescence bias** is the tendency to agree with the interviewer. In general, participants are more likely to agree than to disagree. Thus, the question should not imply that there is a "correct" answer. Instead of asking for agreement or disagreement to a predefined statement, researchers can ask users to explain their point of view.
+
+<details open>
+
+<summary>Example</summary>
+
+<br>
+
+_“Would you agree that you are satisfied with the current interface?”_ This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>
+
+_“How do you feel about the current interface?”_ This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details>
+
+### Confirmation Bias
+
+**Confirmation bias** is the tendency for researchers to interpret data in a way that supports their opinions and beliefs. Researchers know the study’s goal, so they might make connections that aren’t explicit in the data set based on their opinions, beliefs, or expectations on how that goal should be met.
+
+<details open>
+
+<summary>Example</summary>
+
+<br>
+
+This survey asked, “Which of the following emotions do you feel when at the airport? Please select the options below.”
+
+![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png)
```

Comment:
Just want to confirm with review that this is a proper way to link to media content in docs entries.

Applies to all media links below.

Reply:
Hi @mmhughes! :) According to the [File Hosting section of the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting), you'd have to upload them to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).

Reply:
@lucwhite the link should be `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.       The image files should be in the `/media` folder. (The image links will work once the PR is merged.)